contract_sale/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
I migrated according to the guide and there's no such thing for png images.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

Maybe we should update the migration guide so that this is done automatically.

Reply:
There's already a point about that:
```rst
* Update README.rst from https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst if not updated to the latest template.
```